### PR TITLE
Add deploy status message

### PR DESCRIPTION
**Pull request by flenter**
Also includes:
* wercker.yaml with validate-step + shellcheck steps
* implemented shellcheck feedback (minor changes)
* added badge
* updated readme, stating that it requires python 2.x
* bumped version to 1.1.0

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -48,23 +48,24 @@ else
 fi
 
 if [ ! -n "$WERCKER_EMAIL_NOTIFY_BODY" ]; then
-  DEFAULT_BODY="Build $WERCKER_RESULT.\n"
+  DEFAULT_BODY=""
 
   if [ ! -n "$DEPLOY" ]; then
-    DEFAULT_BODY="$DEFAULT_BODY\nSee <$WERCKER_BUILD_URL>\n"
+    DEFAULT_BODY="$DEFAULT_BODY\nBuild $WERCKER_RESULT.\nSee <$WERCKER_BUILD_URL>\n"
```
Comment (hatchan) [inline run.sh]: Know it's not part of this pr, but I would rather see this if statement inverted.

**Comment by hatchan** [comment]
:100:

File: run.sh (hunk 1 above)
Comment (jakirkham) [inline run.sh]: If you make this change, can I ask that it be `Build $WERCKER_RESULT.\n\n`? It makes it a little easier to read having that extra whitespace. Thanks.

File: run.sh [hunk 2]
```diff
@@ -48,23 +48,24 @@ else
...
   else
-    DEFAULT_BODY="See <$WERCKER_DEPLOY_URL>\n"
+    DEFAULT_BODY="$DEFAULT_BODY\Deploy $WERCKER_RESULT\nSee <$WERCKER_DEPLOY_URL>\n"
```
Comment (jakirkham) [inline run.sh]: I see what you were aiming for now. Looks good. Again adding another newline would help `Deploy $WERCKER_RESULT\n\n`. Also, it looks line an `n` got lost here `\Deploy`.

**Comment by jakirkham** [comment]
Put the noted changes into a PR based off of this one. ( https://github.com/wercker/step-email-notify/pull/4 )

**Comment by jakirkham** [comment]
Sorry, have some more minor changes. ( https://github.com/wercker/step-email-notify/pull/5 )

**Comment by flenter** [comment]
The step has been deployed/released

**Comment by jakirkham** [comment]
Thanks @flenter.
